js/basket-signup.js

```diff
@@ -0,0 +1,78 @@
+// Posting to Basket
+// Doc: https://basket.readthedocs.io/newsletter_api.html#news-subscribe
+
+var url = `https://www.mozilla.org/en-US/newsletter/`;
+
+var basketSignup = function(transaction, onSuccessCallback, onFailCallback) {
+  console.log(`basketSignup`, transaction);
+
+  var payload = {
+    format: `H`, // HTML emails
+    newsletter: `mozilla-foundation`,
```

Comment:
hardcoded to `mozilla-foundation` since this is the only newsletter option we have on Pulse